provision/minikube/measure/scale-and-measure-loop.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+export timestamp="${timestamp:-$(date -uIseconds)}"
+
+scale=${1:-1}
```

Comment:
this script could add a section for help to understand the params better, it would skip memory management if its run without any args